it/controllers/CalculationControllerISpec.scala

```diff
@@ -325,6 +337,8 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase with FeatureS
         penaltyStatus = LPPPenaltyStatusEnum.Accruing,
         penaltyAmountPaid = None,
         penaltyAmountOutstanding = Some(123.45),
+        penaltyAmountPosted = 0,
+        penaltyAmountAccruing = 0,
```

Comment:
This (penaltyAmountAccruing) would be 123.45, also I don't think it would have an outstanding amount anymore